### DISCUSSION TITLE
Flash under Home Assistant with EspHome

### DISCUSSION THREAD

**Discussion by titou4307** [athom-tech/esp32-configs#28] [Q&A]
Hello
I am a newbee with this material : 

4CH ESP32 RF322 Inching/Self-lock Relay for ESPHome

I want to flash this relay with esphome under home assistant....

So does I need to solder contact for connect TTL to my Pc ?
For boot what is the button ? 

And third question : what option does I use under Esphome/ha to flash ? 
ESP32-C3 and keep Use recommanded setting ?
or
Choose other ESP32-C3 ?

Thanks

**Comment by tarontop** [comment]
You just need to connect your USB cable and select ESP32, everything is automatic, you don't need to press any buttons

**Comment by dmshimself** [comment]
If you are using the device as delivered, then follow the slip of paper to configure it to use your local wifi and then ESPHOME will spot it after a while and allow you to Adopt it.  Then you can make mods as needed.
